ClLike/cl_like/ept.py

```diff
@@ -131,12 +132,55 @@ def _get_ia_bias(self, pk):
                                      P_window=self.P_window,
                                      C_window=self.C_window)
 
-    def get_pk(self, kind, pgrad=None, cosmo=None, sub_lowk=False):
+    def get_pk(self, kind, pnl=None, cosmo=None, sub_lowk=False, alt=None):
+        # Clarification:
+        # We are expanding the galaxy overdensity as:
+        #   d_g = b1 d + b2 d2^2/2 + bs s^2/2 + bk k^2 d
+        # (see cell 10 in
+        # https://github.com/JoeMcEwen/FAST-PT/blob/master/examples/fastpt_examples.ipynb). # noqa
+        # The `dd_bias` array below contains the following power
+        # spectra in order:
+        #  <d,d^2>
+        #  <d^2,d^2> (!)
+        #  <d,s^2>
+        #  <d^2,s^2> (!)
+        #  <s^2,s^2> (!)
+        # So: the d^2 and s^2 are not divided by 2
+        # Also, the spectra marked with (!) tend to a constant
+        # as k-> 0, which we can suppress with a low-pass filter.
+        # The power spectra provided below are:
+        # mm -> <d*d> (returns pnl)
+        # md1 -> <d*d> (returns pnl)
+        # md2 -> <d*d^2/2>
+        # ms2 -> <d*s^2/2>
+        # mk2 -> k^2 <d*d> (with <d*d> as pnl)
+        # d1d1 -> <d*d> (returns pnl)
+        # d1d2 -> <d*d^2/2>
+        # d1s2 -> <d*s^2/2>
+        # d1k2 -> k^2 <d*d> (with <d*d> as pnl)
+        # d2d2 -> <d^2/2*d^2/2>
+        # d2s2 -> <d^2/2*s^2/2>
+        # d2k2 -> k^2 <d*d^2/2>, not provided
+        # s2s2 -> <s^2/2*s^2/2>
+        # s2k2 -> k^2 <d*s^2/2>, not provided
+        # k2k2 -> k^4 <d*d>, not provided
+        # When not provided, this function just returns `alt`
+
+        kind = kind.replace('m', 'd1')
```

Comment:
What do you expect `kind` to be as input?

Reply:
so `kind` is what's on the left of the arrow?

Reply:
yep